src/execution/values.js

```diff
@@ -165,6 +165,31 @@ export function getArgumentValues(
 }
 
 /**
+ * Prepares an object map of directive name to argument values given a list of
+ * directive definitions and AST node.
+ */
+export function getAppliedDirectives(
```

Comment:
> How about accepting a Schema instance instead? Those define all relevant directives

@leebyron  I needed this function to only get directives relevant for my tool/library so it doesn't make sense to return all of them, e.g. `buildASTSchema` only needs the value of `@deprecated` and doesn't care about the rest of directives.

So I changed it to accept a `GraphQLDirective` as you suggested.
However, I renamed it to `getDirectiveArgs` since I want to keep `getAppliedDirectives` name for the function that you suggested above (accepts `GraphQLSchema`), which in addition to returning arguments will report unknown directives but it's a topic for a separate PR.